modules/crypto/src/lib/utils/base64-utils.js

```diff
@@ -1,22 +1,3 @@
-/**
```

Comment:
Why not move everything here into `digest-utils`? The functions in both files seem to be all related, and it is a small file.

Reply:
The reason I am planning to move the `toBase64` function into loader utils, as an option to the problematic `atob` function.